packages/fx-core/src/common/tools.ts

```diff
@@ -456,7 +456,10 @@ export function canAddSso(
   }
 
   const solutionSettings = projectSettings.solutionSettings as AzureSolutionSettings;
-  if (!(solutionSettings.hostType === HostTypeOptionAzure.id)) {
+  if (
+    !projectSettings.solutionSettings ||
```

Comment:
'Add SSO' is allowed for existing tab app scenario.

Reply:
Thanks, updated.